combineRooms/combineRooms.py

```diff
@@ -0,0 +1,422 @@
+            "gives franchise scouts an opportunity to see how you play. No pressure though! The primary goal for combines is to give "
+            "everybody an opportunity to get introduced to gameplay at their respective tiers."
+
+            "\n\n__Server Information__"
+            "\nServers can be made by anybody in the combine room. We do ask that the lobbies are made with the following naming convention:"
+            "\n\n**Lobby Info:**"
+            "\n - Name: **<tier><room number>**"
+            "\n - Password: **{1}<room number>**"
+            
+            "\n\n**Example:**"
+            "\n - Voice Channel Name: **Challenger room 4**"
+            "\n - Name: **Challenger4**"
+            "\n - Password: **{1}4**"
+
+            "\n\n__The Role of Scouts__"
+            "\n - For lack of a better phrase, scouts are \"in charge\" of running combines."
+            "\n - If a scout requests a lineup, please respect this request."
+            "\n - If a scout requests for mutator settings such as adjusted time length, or a goal limit, please respect this request."
+            "\n - If you have concerns with how combines are being run, contact a mod or an admin."
+
+            "\n\n__Other Notes__"
+            "\n - Please try to curb your particpation in combines towards your own tier. Do not play outside of your tier without being requested "
+            "by a scout, or asking permission of the other players in the combine room."
+            "\n - Don't stress! All players have good and bad days. Scouts care more about _how you play_ than _how your perform_. If you have a "
+            "rough game, or a bad night, you'll have plenty of opportunity to show your abilities in remaining combine games"
+            "\n - As per {2} rules, do not be toxic or hostile towards other players."
+            "\n - GLHF!"
+        ).format(guild.name, acronym.lower(), acronym)
+        await tc.send(info_message)
+    
+    async def _add_combines_voice(self, guild: discord.Guild, tier: str):
+        # user_limit of 0 means there's no limit
+        # determine position with same name +1
+        category = await self._combines_category(guild)
+        tier_rooms = []
+        for vc in category.voice_channels:
+            if tier in vc.name:
+                tier_rooms.append(vc)
+
+        room_makeable = False
+        new_position = None
+        new_room_number = 1
+        while not room_makeable:
+            room_makeable = True
+            for vc in tier_rooms:
+                i = vc.name.index("room ") + 5
+                # DISABLED: room count in name
+                # j = vc.name.index(" (")
+                vc_room_num = int(vc.name[i:])  # j])
+                if vc_room_num == new_room_number:
+                    new_room_number += 1
+                    new_position = vc.position
+                    room_makeable = False
+        
+        ppr = await self._players_per_room(guild)
```

Comment:
I'd comment this line out as well since it's not being used in the room names any more.

Reply:
Done